src/functions/csvJSON.js

```diff
@@ -1,20 +1,11 @@
-export function csvJSON(csv) {
+export const csvJSON = csv => {
```

Comment:
please rename to `parseCsv` which better describes what it does

Reply:
4b94bd6319b9bab01b00f52ce53f4b72ed2502ae